frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelWelcome.tsx

```diff
@@ -0,0 +1,115 @@
+import { IconCursorClick, IconGear, IconHome, IconInfo, IconNight, IconNotebook, IconSearch } from '@posthog/icons'
+import { LemonButton, Link } from '@posthog/lemon-ui'
+import { useActions } from 'kea'
+import posthog from 'posthog-js'
+import { useEffect } from 'react'
+
+import { KeyboardShortcut } from '../../components/KeyboardShortcut'
+import { sidePanelStateLogic } from '../sidePanelStateLogic'
+
+const blogPostUrl = 'https://posthog.com/blog/why-redesign'
+
+export const SidePanelWelcome = (): JSX.Element => {
+    const { closeSidePanel } = useActions(sidePanelStateLogic)
+
+    useEffect(() => {
+        return () => {
+            // Linked to the FF to ensure it isn't shown again
+            posthog.capture('3000 welcome acknowledged', {
+                $set: {
+                    [`3000-welcome-acknowledged`]: true,
+                },
+            })
+        }
+    }, [])
+
+    return (
+        <div className="flex flex-col items-center justify-center m-4 my-10 flex-1">
+            <h1 className="font-bold text-4xl text-center">
+                Welcome to <span className="text-primary-3000">PostHog&nbsp;3000</span>
+            </h1>
+
+            <p className="max-w-120 text-center">
+                <b>PostHog 3000</b> is the codename for our revamped user experience. You can{' '}
+                <Link to={blogPostUrl}>read more about it here</Link> but the headline features you will notice are:
+            </p>
+            <ul className="space-y-4">
+                <li className="bg-bg-3000 border rounded p-4 flex gap-4 items-center">
+                    <IconNight className="text-3xl" />
+                    <div className="flex-1">
+                        <h3 className="mb-0 font-semibold">Colors, data density, dark mode</h3>
```

Comment:
The default browser h3 font size feels a bit off in 3000… I would suggest setting `text-base` here, but actually can we just set a `font-size: 1rem` override under `.posthog-3000` in `global.scss`?